.travis.yml

```diff
@@ -91,6 +94,8 @@ jobs:
     - <<: *linux-build
       env:
         COMPILER_VERSION=4.9 MAKE_TARGET=$MAKE_TARGET:deb
+      before_cache:
+        - if [[ -n $TRAVIS_TAG ]]; then cp pkg/deb/*.deb $CACHE_DIR/; fi
```

Comment:
This probably needs quotes: `[[ -n "$TRAVIS_TAG" ]]` otherwise if the tag is empty it could give a syntax error (at least on some shell implementations).